packages/capabilities/test/capabilities/access.test.js

```diff
@@ -3,6 +3,8 @@ import { access } from '@ucanto/validator'
 import { Verifier } from '@ucanto/principal/ed25519'
 import * as Access from '../../src/access.js'
 import { alice, bob, service, mallory } from '../helpers/fixtures.js'
+import * as Ucanto from '@ucanto/interface'
```

Comment:
This would have required a comment to disable tsc no-unused-vars, but it no longer does because of [adding jsdoc/no-undefined-types rule](https://github.com/web3-storage/w3protocol/pull/409/files#diff-ba437578db4413909c1b02066b4d818ccb40dc79932f754051415b17f39a150aR92)

related to: https://github.com/web3-storage/w3protocol/pull/392/files#r1090596726